frontend/src/shared/file.utils.ts

```diff
@@ -1066,6 +1067,38 @@ export function getRankingStageCSVColumns(
         : '',
   );
 
+  // 🧩 Leadership Rejection (LR) extension
+  if (participant) {
+    // Column data
+    let leaderStatus = '';
+    let winnerId = '';
+    if (publicData && 'leaderStatusMap' in publicData) {
+      const lrData = publicData as LRRankingStagePublicData;
+      const participantPublicId = participant.profile.publicId ?? '';
+      leaderStatus = lrData.leaderStatusMap?.[participantPublicId] ?? '';
+      winnerId = lrData.winnerId ?? '';
+    }
+    console.debug(
+      '[LR][getRankingStageCSVColumns] [Leadership Rejection (LR) extension] columns data: leaderStatus=',
+      leaderStatus,
+      ', winnerId=',
+      winnerId,
+    );
+    // Column for participant’s leader status
+    columns.push(leaderStatus);
```

Comment:
In both branches of this if/else, the code pushes two columns. It's good that they're balanced. I'm a a little concerned though about the risk of adding columns for code that may not be expecting new columns. That is, I don't know what downstream code consumes the output of `getRankingStageCSVColumns()` but I'd want to make sure that that code is resilient to changing the returned number of columns. (I would hope that downstream code handles this fine, I just don't know for sure.)